packages/client/src/utils/referenceApi.ts

```diff
@@ -134,6 +142,7 @@ export interface IApplicationConfig {
   LOGIN_BACKGROUND: ILoginBackground
   USER_NOTIFICATION_DELIVERY_METHOD: string
   INFORMANT_NOTIFICATION_DELIVERY_METHOD: string
+  SEARCH_DEFAULT_CRITERIA: SearchCriterias
```

Comment:
It's better to have this as an optional property. That way it would stay as backwards compatible